TACC/Core-Portal
# task/TV3-56: Interactive app infrastructure

## Overview

Infrastructure to support interactive apps in Tapis v3

## Related

* [TV3-56](https://jira.tacc.utexas.edu/browse/TV3-56)
* [TV3-112](https://jira.tacc.utexas.edu/browse/TV3-112)

## Changes



## Testing

1. In the admin panel, add the app `matlab-9.13.0`
2. Ensure you add a matlab "license" for yourself in the admin panel
3. For good measure, add yourself to the [matlab unix group](https://tas.tacc.utexas.edu/TACCAdministration/Accounts/UnixGroupEdit.aspx?id=71088), which bypasses the need for a license
4. Launch the app [matlab-9.13.0](https://cep.test/workbench/applications/matlab?appVersion=9.13.0)
5. ???
6. Profit

## UI



## Notes
